gluster_peers_bricks.yml

```diff
@@ -0,0 +1,12 @@
+---
+#
+# Create gluster trusted storage pool and bricks
+# on all available devices on gluster nodes.
+#
+
+- hosts: gluster
+  remote_user: root
+  roles:
+  #- { role: qe-gluster-peers, gluster_nodes: "{{groups['gluster']}}"}
```

Comment:
Based on the description, it should be possible to:

> configure gluster trusted storage pool and bricks without Tendrl server

But for this to work, we would need to add `gluster-centos` role there.